applications/allocation_result_mapper.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from allocation.allocation_solver import AllocatedEvent
+from applications.models import (
+    ApplicationEventResult,
+    ApplicationEventSchedule,
+    ApplicationEventScheduleResult,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AllocationResultMapper(object):
+    def __init__(self, allocated_events: [AllocatedEvent]):
+        self.allocated_events = allocated_events
+
+    def to_events(self):
+        for allocated_event in self.allocated_events:
+            try:
+                res, created = ApplicationEventResult.objects.update_or_create(
+                    application_event_id=allocated_event.event_id,
+                    defaults={
+                        "allocated_reservation_unit_id": allocated_event.space_id
+                    },
+                )
+            except Exception:
+                logger.exception(
+                    "AllocationResultMapper: error occurred while creating event results."
+                )
+
+            try:
+                application_event_schedule = ApplicationEventSchedule.objects.get(
+                    pk=allocated_event.occurrence_id  # Correct?
```

Comment:
Yeah, this should be correct. Need to do something about the naming conventions to make them more clear, like we use occurence and space in allocation models, and something different elsewhere, so it's a bit confusing.